src/jdk.jfr/share/classes/jdk/jfr/events/SelectorSelectEvent.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package jdk.jfr.events;
+
+import jdk.jfr.*;
+import jdk.jfr.internal.MirrorEvent;
+import jdk.jfr.internal.Type;
+
+@Name(Type.EVENT_NAME_PREFIX + "SelectorSelect")
+@Label("SelectorSelect")
+@Category("Java Application")
+@Description("Selector Select Operation")
+@MirrorEvent(className = "jdk.internal.event.SelectorSelectEvent")
+public class SelectorSelectEvent extends AbstractJDKEvent {
+
+    @Label("SelectionKey Count")
+    @Description("Number of channels ready for I/O or added to ready set")
+    public int selectionKeyCount;
+
+    @Label("Timeout Value")
+    @Description("Milliseconds to block, or zero to block indefinitely")
```

Comment:
The unit should not be mentioned. A GUI will typically convert it from the TimeSpan information and it will just look confusing if the descriptions says milliseconds when it's, for instance, displayed as 1 s.